src/css/application.css

```diff
@@ -113,13 +113,15 @@ body {
 /** @define top-bar */
 
 .top-bar {
+  align-items: center;
   background-color: var(--color-green);
+  color: rgba(0, 0, 0, 0.75);
   display: flex;
   flex: 0 0 auto;
-  font-size: 2vh;
+  font-family: sans-serif;
```

Comment:
What’s the thinking here? I would expect this to be one of the project fonts, or nothing at all?

Reply:
It was part of my original mockup. In general, it's a little wider and easier to read.

![21slczmmuh](https://user-images.githubusercontent.com/3432105/32761626-5cc1dff6-c8c2-11e7-8b84-c9b3c7fbaa6c.gif)